default/scripting/species/SP_SLY.focs.txt

```diff
@@ -24,7 +24,7 @@ Species
 
         [[BAD_POPULATION]]
```

Comment:
Instead of using the standard population effectsgroup macros and needing to adjust gas giant size to balance just this species, could give them a custom alternative / lower population mechanic...

Reply:
Would like to do that in the next iteration. I'd prefer more playtesting results first.

Reply:
> Would like to do that in the next iteration. 

Both Vezzra and Geoff have now raised concerns (which make sense to me) about the whole approach of trying to balance the Sly by changing the more general characteristic of gas giant size.   That doesn't seem like a suitable situation for making a change while planning to undo it in the near future. 

So, it seems to me that shifting the Sly from Bad Population to a new Very Bad Population with factor -0.5 would accomplish very nearly the same thing (subject perhaps to what priority it got), and that would be a better thing to try here.

Reply:
I was thinking they could be giving a completely separate population mechanic... like fixed 5 max pop, and not fit into the good/average/bad system.

Reply:
with your suggestion none of the growth techs would mean anything. that doesnt seem right.
hm i think for such things we'll need a design discussion in the forum. for this iteration i will do something analog to the habitable size 4